### DISCUSSION TITLE
what is the different between timestampz and timestamp with timezone

### DISCUSSION THREAD

**Discussion by hiepxanh** [pgmodeler/pgmodeler#1706] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/16970990/183041634-f0394de2-6367-4ec9-9d25-794483ba1726.png)

here it is, I'm not really understand timestampz so I choice timestamp with timezone, is that any different :D

**Comment by rkhaotix** [comment, marked as answer]
```timestamptz``` is just an alias to ```timestamp with time zone``` , so they are just the same data type. :)

**Reply by hiepxanh** [reply]
Ohhhh so is that safe to replace all 'timestamp with timezone' by 'timestampz' sir?

**Reply by rkhaotix** [reply]
Sure. It's safe.
